doc/tdshim_spec.md

```diff
@@ -626,7 +626,12 @@ memory before transfer to a payload.
 
 If a TD-memory region is unaccepted memory and requires TDCALL
 [TDG.MEM.PAGE.ACCEPT] in the TD guest OS, then the TD Shim shall have the final
-memory map report this region with AddressRangeUnaccepted.
+memory map report this region with AddressRangeMemory and build a bitmap for
+all the system memory to mark whether each 2MB size page has been accepted or
+not. A bit set to 1 in bitmap indicates that the corresponding page has not
+been accepted yet. For the payload supporting Linux Boot Protocol, the TD Shim
+shall write the 64-bit bitmap address to boot parameter
+([offset 0x78](https://github.com/intel/tdx/blob/guest-unaccepted-memory/Documentation/x86/zero-page.rst)).
```

Comment:
Please describe the memory type of the bitmap as well. (e.g. normal memory, or reserved)

Reply:
Sure